framework/src/preconditioners/MoosePreconditioner.C

```diff
@@ -27,7 +27,7 @@ InputParameters validParams<MoosePreconditioner>()
   InputParameters params = validParams<MooseObject>();
   params.addPrivateParam<FEProblemBase *>("_fe_problem_base");
 
-  MooseEnum pc_side("left right symmetric", "right");
+  MooseEnum pc_side("left right symmetric");
```

Comment:
how about `none`? In PETSc we have `left right symmetric` but not `none`. If `none` is set, we let PETSc make the decision.